torch/csrc/jit/script/compiler.h

```diff
@@ -17,6 +17,14 @@ struct CallsiteDescriptor {
   bool allow_varargs;
 };
 
+struct DefAndTypes {
```

Comment:
suggestion: `TypedDef`?

Reply:
Also it would be nice if we could just put the types in the `Def`. In Python 3 syntax they really are part of the definition, it's just that our parser doesn't support type annotations (I think)